src/node_crypto.cc

```diff
@@ -3797,9 +3797,7 @@ void Hmac::HmacDigest(const FunctionCallbackInfo<Value>& args) {
 
   enum encoding encoding = BUFFER;
   if (args.Length() >= 1) {
-    encoding = ParseEncoding(env->isolate(),
-                             args[0]->ToString(env->isolate()),
-                             BUFFER);
+    encoding = ParseEncoding(env->isolate(), args[0], BUFFER);
```

Comment:
@addaleax You are right, I will add the `CHECK`s and add the regression test.

> maybe you could fix that up and drop the unused code bits?

Is it okay to do that in a separate PR or will that create too much organizational overhead?